blog/content/post/2021-06-16-OSF_vPub_02.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "OSF vPub (vBeer) - what a fun event!"
+abstract: "Our 2nd OSF vPub gathering was really fun and interesting
+          - together with Richard Stallman as our honored guest!"
+cover: /covers/OSF_vPub.png
+author: mike.banon
+layout: post
+published: true
+date: 2021-06-16
+archives: "2021"
+
+tags:
+  - Conference
+  - Event
+  - vPub
+  - Firmware
+  - Open-source
+categories:
+  - Firmware
+  - Miscellaneous
+
+---
+
+On our 2nd OSF vPub, amoung the many major topics we have explored the software
+freedom of RISC-V implementations and new devices such as BeagleV. RISC-V is
+attractive to the people who aren't satisfied with the software freedom problems
+of the more popular architectures such as x86. And it's essential to ensure that
+this emerging platform is going in the right way and figure out how can we help.
+
+Among our many close friends from all over the world who found the time to join
+us, it was a great honour to be visited by Richard Stallman - who shared lots of
+worthy ideas (such as having a roadmap & status pages) to assist the liberation
+of RISC-V. Many people (myself included!) were happy to discuss with RMS, - and
+we at 3mdeb will try to invite more such prominent guests to our future events.
```

Comment:
@mikebdp2 maybe we can elaborate discussed topics more? This summary is very short and does not reflect amount of topics we discused.

Reply:
What I miss here:
1. Link to events.3mdeb.com to vPub event - it is not yet there, but should be @artur-rs cc
2. Events have their own numbers and names as in slides
3. We poorly leverage chat history and there are real perks there
4. We would like to introduce project announcement board, anyone who want to discuss his/her project related to OSF/OSHW/Open ISA is welcome
5. We would like to mention data of next event which should happen around August - this post do not mention even when the event took place.
6. Links to social media announcements would be appreciated
7. Adding event poster would also be great

Reply:
@pietrushnic Sorry for delay. With a commit below I think I addressed 2, 3 (to some extent), 4 and 7. Regarding the other points: 1 - waiting for @artur-rs , 5 - don't know the exact date of next event (we should discuss it), 6 - need a clarification, links to which announcements I should include. Also, 4 ("We would like to introduce project announcement board") could be addressed better if 3mdeb could provide a more high-level "project announcement board" than my simple invitation note.

Reply:
@mikebdp2 we've added the event: https://3mdeb.com/events/#vpub-21 but there is no content yet

Reply:
@mikebdp2 "project announcement board" would be time slot dedicated to anyone who would like to shout out about his/her OSF project. Last vPub there were couple people who join just because of that. vPub has good audience for that purpose. I'm not sure what to say more about that. Announcements should be kept short (max 5-10min and then discussion), should provide project name and some link to follow up. We are quite flexible about rules. I'm pretty sure you feel what I want to say and eventually I can review whatever you will came with.